test/rosenbrock.jl

```diff
@@ -147,6 +147,7 @@ sol = solve(prob, NLO(:G_MLSL_LDS), local_method = NLO(:LD_LBFGS), local_maxiter
 # @test 10*sol.minimum < l1
 
 using Evolutionary
+prob = GalacticOptim.OptimizationProblem(optprob, x0)
```

Comment:
Just curious, why was this needed?

Reply:
I discovered that previously the `__solve` function ignored the `lb` and `ub` of the `OptimizationProblem` but now it recognises them and passes them to the `Evolutionary` optimisers. Hence to keep the test the same and not failing it I redefined the problem w/o bounds. I hope that made sense.